vignettes/profiling.Rmd

```diff
@@ -0,0 +1,36 @@
+---
+title: "Profiling Stan programs with CmdStanR"
+author: "Rok Češnovar and Jonah Gabry"
+output: 
+  rmarkdown::html_vignette:
+    toc: true
+    toc_depth: 4
+params:
+  EVAL: !r identical(Sys.getenv("NOT_CRAN"), "true")
+vignette: >
+  %\VignetteIndexEntry{Profiling Stan programs with CmdStanR}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
```

Comment:
I just copied all this from the other vignettes (but I put your name first in the authors)